### PR TITLE
Fix eslint errors

### PR DESCRIPTION
**Pull request by JoeCortopassi**
go through the `new/` and `generate/` directories to fix errors discovered by the new eslint rules

### CODE REVIEW

File: new/Index.js [hunk 1]
```diff
@@ -7,6 +7,11 @@ import "assets/css/normalize.css";
  * should not remove those or the application will likely stop working.
  */
 export default class Index extends Component {
+  static propTypes = {
+    head: PropTypes.any,
+    body: PropTypes.any
+  }
```
Comment (toddw) [inline new/Index.js]: static properties should have `;`

File: new/src/components/MasterLayout.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 /* @flow */
-import React, { Component } from "react";
-import { Link } from "react-router";
+import React, { Component, PropTypes } from "react";
 
 export default class MasterLayout extends Component {
+  static propTypes = {
+    children: PropTypes.any
+  }
```
Comment (toddw) [inline new/src/components/MasterLayout.js]: `;` and blank line after

**Comment by toddw** [comment]
> go through the new/ and generate/ directories to fix errors discovered by the new eslint rules

**Comment by toddw** [comment]
> go through the new/ and generate/ directories to fix errors discovered by the new eslint rules

I'm not seeing any of the files in the `generate` directory

**Comment by christinebrass** [comment]
I'm seeing these errors still with our latest eslintrc:

```
› eslint new generate

/opt/truecar/src/gluestick/new/src/config/application.js
  13:71  error  Missing semicolon  semi

/opt/truecar/src/gluestick/generate/component.test.js
  7:42  error  Unexpected use of undefined  no-undefined

/opt/truecar/src/gluestick/generate/container.test.js
  7:42  error  Unexpected use of undefined  no-undefined

/opt/truecar/src/gluestick/generate/reducer.test.js
  6:15  error  Unexpected use of undefined  no-undefined

✖ 4 problems (4 errors, 0 warnings)
```

**Comment by JoeCortopassi** [comment]
Got 'em @toddw @cybrass :+1:

File: generate/reducer.test.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import reducer from "reducers/__$NAME__";
 describe("reducers/__$NAME__", () => {
   it("returns the initial state", () => {
     expect(
-      reducer(undefined, {})
+      reducer(void 0, {})
```
Comment (christinebrass) [inline generate/reducer.test.js]: Clever! In terms of readability, what do you think of this?

```
it("returns the initial state", () => {
    let state;
    expect(
      reducer(state, {})
    ).to.equal(null);
  });
```

**Comment by christinebrass** [comment]
:+1:

File: generate/reducer.test.js (hunk 3 above)
Comment (JoeCortopassi) [inline generate/reducer.test.js]: :+1:

**Pull request merged**